ui/pages/permissions-connect/snaps/snap-update/snap-update.js

```diff
@@ -193,10 +199,11 @@ export default function SnapUpdate({
                 revokedConnections={revokedConnections}
                 newConnections={newConnections}
                 targetSubjectMetadata={targetSubjectMetadata}
+                showAllPermissions={onShowAllPermissions}
               />
             </Box>
-            {isScrollable && !isScrolledToBottom ? (
-              <Box className="snap-update__scroll-button-area">
+            <Box className="snap-update__scroll-button-area">
+              {isScrollable && !hasScrolledToBottom && !showAllPermissions ? (
```

Comment:
Same question as in snap-install

Reply:
I think that we had an agreement on this, so we show the arrow only if there are lot of permissions to scroll at first, then if user clicks on the arrow down or reveal all permissions, we do not need to display the arrow anymore, also confirm button is enabled in case where user reviewed the most important permissions.